certified-connectors/RecordedFutureSandbox/README.md

```diff
@@ -0,0 +1,36 @@
+The Recorded Future Sandbox Connector enables security and IT teams to analyze and understand URLs, which provides safe and immediate behavioral analysis of URLs, helping contextualize key artifacts in an investigation, leading to faster triage. Through this connector, organizations can incorporate the malware analysis sandbox into automated workflows with applications such as Microsoft Defender for Endpoint and Microsoft Sentinel.
```

Comment:
@lindarecfut please include all required sections in the readme file. Please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

Reply:
Thanks, have made a new commit with changes now.